spec/rails_helper.rb

```diff
@@ -27,6 +27,8 @@
   config.include Devise::Test::ControllerHelpers, type: :controller
   config.extend ControllerMacros, type: :controller
 
+  MiniRacer::Platform.set_flags! :single_threaded
```

Comment:
May I know what is this for?

Reply:
This is to silence the warnings

`WARNING: V8 isolate was forked, it can not be disposed and memory will not be reclaimed till the Ruby process exits.
It is VERY likely your process will hang.
If you wish to use v8 in forked environment please ensure the platform is initialized with:
MiniRacer::Platform.set_flags! :single_threaded`

Reply:
this is meant to be for the test environment right? making the the miniracer single_threaded that is

Reply:
Yup, it only affects the test environment